### PR TITLE
Small Journal CSS fixes

### PR DESCRIPTION
**Pull request by aumayr**
- Don't allow date or numbers to wrap
- Fix black separator square width

### CODE REVIEW

**Comment by aumayr** [comment]
Another point for discussion: In the Journal table, the payee and narration are vertically centered, as well as tags and the points for metadata entries. The date, flag and numbers are not vertically centered, which looks weird. 

Would be vertically centering everything in the Journal table be a good solution?

File: fava/static/css/journal-table.css [hunk 1]
```diff
@@ -136,6 +137,7 @@
 
 .journal .datecell {
   width: 5.5rem;
+  white-space: nowrap;
```
Comment (yagebu) [inline fava/static/css/journal-table.css]: Shouldn't the date cell always have enough space?

Comment (aumayr) [reply fava/static/css/journal-table.css]: In Chrome and Safari sometimes it doesn't.

Comment (yagebu) [reply fava/static/css/journal-table.css]: Does it look bad with the nowrap setting then? We could also make this cell minimally wider

**Comment by yagebu** [comment]
Noticed the vertical centering too

> Would be vertically centering everything in the Journal table be a good solution?

I guess

File: fava/static/css/journal-table.css (hunk 1 above)
Comment (yagebu) [reply fava/static/css/journal-table.css]: :+1:

**Comment by yagebu** [comment]
Since the vertical centering might not be as trivial as these changes here, I've merged this already.

**Comment by aumayr** [comment]
> Since the vertical centering might not be as trivial as these changes here, I've merged this already.

Are you having a go at this?

**Comment by yagebu** [comment]
> Are you having a go at this?

No, I thought you are?

**Pull request merged**